e2e/cypress/integration/accessibility/accessibility_nav_diff_regions_spec.js

```diff
@@ -100,19 +100,16 @@ describe('Verify Quick Navigation support across different regions in the app',
 
     it('MM-T1460_5 Verify Navigation Support in LHS Sidebar', () => {
         // # Change the focus to Main Menu button
-        cy.get('#headerInfo button').focus().tab({shift: true}).tab();
-
-        // * Verify nav support in LHS sidebar header
-        verifyNavSupport('#lhsHeader', 'team menu region', '5');
```

Comment:
I don't see this supported with in the new sidebar presentation with Global header enabled.